Doc/library/socket.rst

```diff
@@ -77,6 +77,11 @@ created.  Socket addresses are represented as follows:
   backward compatibility.  Note, however, omission of *scopeid* can cause problems
   in manipulating scoped IPv6 addresses.
 
+  .. versionchanged:: 3.6
```

Comment:
I don't know which version to write here.

Reply:
3.6 is in bugfix mode.
The PR should land into 3.7 only IMHO

Reply:
This is 3.7 only.